operator/cmd/mesh/profile-common.go

```diff
@@ -214,6 +221,50 @@ func genIOPSFromProfile(profileOrPath, fileOverlayYAML, setOverlayYAML string, s
 	return util.ToYAMLWithJSONPB(finalIOPS), finalIOPS, nil
 }
 
+// translateK8SfromValueToIOP use reverse translation to convert k8s settings defined in values API to IOP API.
+func translateK8SfromValueToIOP(userOverlayYaml string, l *Logger) (string, error) {
```

Comment:
maybe add: "This ensures that user overlays that set k8s through spec.values are not overridden by spec.components.X.k8s settings in the base profiles."

Reply:
done